subprojects/ide/ide.gradle

```diff
@@ -30,6 +30,7 @@ dependencies {
     compile libraries.inject
 
     testCompile libraries.xmlunit
+    testCompile libraries.equalsVerifier
```

Comment:
This is a nit-picky thing, but we only use this in two tests and I think we could capture about the same thing by checking that the effect of a bad equals/hashCode method is caught in the integration tests (for the generated Eclipse metadata files).